src/pages/home.js

```diff
@@ -83,6 +85,18 @@ export function Home() {
       : dispatch(deselectBenefitActionCreator(benefitId));
   };
 
+  if (fetchBenefitsFailed || fetchBenefitsCountFailed) {
```

Comment:
Not-blocking: Is this going to get lengthily if more error types are captured? If so, should a more generic handler be looked at?

Reply:
We can look at that down the road if need be. Think this works for now

Reply:
Agreed